.github/workflows/tests-ci.yml

```diff
@@ -59,7 +59,8 @@ jobs:
         run: pnpm --recursive --stream --sequential build:tests:node
 
       - name: Run tests for all packages
-        run: pnpm --recursive --stream exec c8 mocha -- --color --reporter mocha-junit-reporter --reporter-options mochaFile=./results.xml
+        # run: pnpm --recursive --stream exec c8 mocha -- --color --reporter mocha-junit-reporter --reporter-options mochaFile=./results.xml
```

Comment:
I think in the future if you just delete `--reporter mocha-junit-reporter --reporter-options mochaFile=./results.xml` you can still debug without switching to the npm command but I aim to have the fix PR up today for a reference for @finn-tbd